example/ddceler/steeringFile.py

```diff
@@ -0,0 +1,59 @@
+"""
+
+Minimal proof of concept for DDG4 steering file to run Geant4 with celeritas integration.
+
+Usage:
+
+ddsim --compactFile=$PWD/example/ddceler/SiD_ConstantField.xml --steering $PWD/example/ddceler/steeringFile.py --outputFile output.root
+
+"""
+
+from DDSim.DD4hepSimulation import DD4hepSimulation
+
+RUNNER = DD4hepSimulation()
```

Comment:
Does DD4hep deliberately not follow [PEP8 style conventions](https://peps.python.org/pep-0008/)?

- Variables like RUNNER should be lowercase
- Functions should be `snake_case`
- Indentation should be 4 spaces